### PR TITLE
Black Friday pricing banner

### PR DESCRIPTION
**Pull request by peratik**
https://www.notion.so/santiment/Black-friday-Pricing-page-9c0e1ba11c96496fb847c7d5422276ce

Day and night mode is the same. so we hardcoded color codes rather than using color vars.

I still believe it is better to convert banners to slots and fill / control them from app.

image

image

image

### CODE REVIEW

File: src/ui/Pricing/Page/BlackFridayBanner.svelte [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+<section class="row justify v-center border body-2 relative {className}">
+  <div class="text">
+    <div class="title">Buy this bear market dip</div>
+    <div class="value">35% Off</div>
+    <div class="condition">for all subscriptions till Nov 30th</div>
+    <a class="btn btn-1 body-1 mrg--b mrg-l" href="https://santiment.net/discord"
+      >Claim your promo code!</a
+    >
+    <div class="note">You will find your promo code in our dicord channel</div>
+  </div>
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/BlackFridayBanner.svelte:17]: Use semantic tags

File: src/ui/Pricing/Page/BlackFridayBanner.svelte [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    height: 100%;
+  }
+
+  .text {
+    max-width: 380px;
+
+    a {
+      display: inline-flex;
+      --v-padding: 9.6px;
+    }
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/BlackFridayBanner.svelte:34]: Use classname instead

File: src/ui/Pricing/Page/BlackFridayBanner.svelte [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  }
+  section {
+    max-width: 100%;
+    width: 1140px;
+    border: 1px solid #222639;
+    border-radius: 8px;
+    padding: 40px 0 40px 72px;
+    margin: 0 auto 80px;
+    overflow: hidden;
+    background-color: #1a1d2f;
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/BlackFridayBanner.svelte:44]:
```suggestion
    background: #1a1d2f;
```

File: src/ui/Pricing/Page/BlackFridayBanner.svelte [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    letter-spacing: 0.01em;
+    color: #ffffff;
+    margin-bottom: 28px;
+  }
+
+  .note {
+    font-style: italic;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 26px;
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/BlackFridayBanner.svelte:76]: User webkit's class

**Review by DmitriVanGuard — CHANGES_REQUESTED** [review]

File: src/ui/Pricing/Page/BlackFridayBanner.svelte (hunk 1 above)
Comment (peratik) [reply src/ui/Pricing/Page/BlackFridayBanner.svelte:17]: What do you suggest for them? h1...h6 ?

File: src/ui/Pricing/Page/BlackFridayBanner.svelte (hunk 4 above)
Comment (DmitriVanGuard) [reply src/ui/Pricing/Page/BlackFridayBanner.svelte]: Component's design has Design System Typography, which is available in webkit.

File: src/ui/Pricing/Page/BlackFridayBanner.svelte (hunk 1 above)
Comment (DmitriVanGuard) [reply src/ui/Pricing/Page/BlackFridayBanner.svelte:18]: Use the ones you think fit the best. I will review once it's ready

**Review by DmitriVanGuard — CHANGES_REQUESTED** [review]
Tablet screenshot is missing

**Comment by peratik** [comment]
> Tablet screenshot is missing

added

File: src/ui/Pricing/Page/BlackFridayBanner.svelte [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+<section class="row justify v-center border body-2 relative {className}">
+  <img src={blackFriday} alt="Black Friday Bear" />
+  <article>
+    <h4 class="h4 title">Buy this bear market dip</h4>
+    <h3>35% Off</h3>
+    <h4 class="h4 condition">for all subscriptions till Nov 30th</h4>
+    <a class="btn btn-1 body-1 mrg--b mrg-l" href="https://santiment.net/discord"
+      >Claim your promo code!</a
+    >
+    <em class="body-2">You will find your promo code in our dicord channel</em>
```
Comment (DmitriVanGuard) [inline src/ui/Pricing/Page/BlackFridayBanner.svelte:17]:
```suggestion
    <p class="body-2">You will find your promo code in our dicord channel</p>
```

**Review by DmitriVanGuard — APPROVED** [review]

**Pull request merged**